src/Umbraco.Web/PublishedContentQuery.cs

```diff
@@ -56,6 +58,13 @@ public IPublishedContent TypedContent(int id)
                 : _typedContentQuery.TypedContent(id);
         }
 
+        public IPublishedContent TypedContent(Guid id)

```

Comment:
Again I know this is obvious, but for docs & intellisense it will help to describe how the IPublishedContent is getting it from the Cache  or if coming back null then it fetches from other?! cache or is it straight from the DB?!

Simple one liner may help so dev's explaining that may how content is fetched when they query using this.